src/content/api/loyalty-program-memberships.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Loyalty Program Memberships
+description: Loyalty Program Memberships model and related endpoints
+nav:
+  path: API/Loyalty Programs
+  order: 2
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Error from '../../components/Error.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+A Loyalty Program Membership is a record of a user being onboarded to a Loyalty Program.
+
```

Comment:
Thoughts on the description of the Memberships ? :)

Reply:
`A Loyalty Program Membership represents an Account's membership to a Loyalty Program`